### PR TITLE
multi batch jobs

### PR DESCRIPTION
**Pull request by mbr84**
When using bulk api, split records into batches containing <= 10000 records, and associate these with a single job

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,23 +18,15 @@
 
 here = os.path.abspath(os.path.dirname(__file__))
 
-about = {}
-with open(os.path.join(here, 'simple_salesforce', '__version__.py'), 'r') as f:
-    exec(f.read(), about)
-
 
 setup(
-    name=about['__title__'],
```
Comment (andscoop) [inline setup.py]: Remove your fork specific details

Comment (mbr84) [reply setup.py]: sorry my fault. I have to get a version of this running on a project for work. The latest commit was meant for another repo. Should be fixed now

**Comment by mbr84** [comment]
@andscoop in case it makes a difference, it can do this now with one call...
screen shot 2018-01-10 at 7 38 35 am

**Comment by jankatins** [comment]
There was just something similar merged, this can probably be closed

**Comment by jon-wobken** [comment]
Solved by #392

**Pull request closed without merging**